portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,8 +14,16 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.FetchOptions;
```

Comment:
Nit: Something we haven't talked about before (My bad I forgot), but usually inputs are sorted in alphabetical order.
For example: FetchOptions should go before PreparedQuery.

Reply:
Style guide: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing